### PR TITLE
Update capability docs to focus on users

### LINKED ISSUE #303: Update Linux Capability Docs
## Which Document page:

On what page is the problem?
security_options.rst

### Expected results:

What are you expecting to see?
How users can manage the capabilities they are granted with `--add-caps` and `--drop-caps`.

### Actual results:

What is the problem, what's misspelled, or what's missing?
Currently, this page documents how admins can grant users various capabilities. These examples should be in the admin docs instead

### PR DESCRIPTION
**Pull request by ikaneshiro**
Signed-off-by: Ian Kaneshiro <iankane@umich.edu>

## This fixes or addresses the following GitHub issues:

- Fixes #303

### CODE REVIEW

File: security_options.rst [hunk 1]
```diff
@@ -16,74 +16,96 @@ Linux Capabilities
...
-    CAP_NET_RAW
+    $ singularity exec --add-caps CAP_NET_RAW library://sylabs/tests/ubuntu_ping:v1.0 ping -c 1 8.8.8.8
+    WARNING: not authorized to add capability: CAP_NET_RAW
+    ping: socket: Operation not permitted
 
-To take advantage of this new capability, the user ``pinger`` must also request
-the capability when executing a container with the ``--add-caps`` flag like so:
+
+Another scenario which is atypical of shared resource environments, but useful
+cloud-native architectures is dropping capabilities when spawning containers as
```
Comment (dtrudg) [inline security_options.rst]:
```suggestion
in cloud-native architectures is dropping capabilities when spawning containers as
```

File: security_options.rst [hunk 2]
```diff
@@ -16,74 +16,96 @@ Linux Capabilities
...
+the root user in order to help minimize attack surfaces. With a default
```
Comment (dtrudg) [inline security_options.rst]:
```suggestion
the root user to help minimize attack surfaces. With a default
```

File: security_options.rst [hunk 3]
```diff
@@ -16,74 +16,96 @@ Linux Capabilities
...
 -----------------------------
 Building encrypted containers
 -----------------------------
 Beginning in Singularity 3.4.0 it is possible to build and run encrypted
-containers.  The containers are decrypted at runtime entirely in kernel space, 
-meaning that no intermediate decrypted data is ever present on disk or in 
-memory.  See :ref:`encrypted containers <encryption>` for more details.
+containers.  The containers are decrypted at runtime entirely in kernel space,
+meaning that no intermediate decrypted data is ever present on disk or in
+memory. See :ref:`encrypted containers <encryption>` for more details.
```
Comment (dtrudg) [inline security_options.rst]: Want to drop the `in memory` here while you are touching this, so the re-wrapping doesn't conflict with me doing it in a separate PR? Or I can do it separately.

**Review by dtrudg — CHANGES_REQUESTED** [review]

File: security_options.rst (hunk 3 above)
Comment (ikaneshiro) [reply security_options.rst]: Yea ill go ahead and do that

**Review by dtrudg — APPROVED** [review]

**Pull request merged**